app/javascript/components/MockSongs.js

```diff
@@ -0,0 +1,105 @@
+let randomSongs = [
```

Comment:
Not a dealbreaker, but since this is mock data that won't change, `const` would probably be more appropriate.